odoo/addons/base/wizard/base_partner_merge.py

```diff
@@ -115,8 +115,9 @@ def _update_foreign_keys(self, src_partners, dst_partner):
         # this guarantees cache consistency
         self.env.invalidate_all()
 
+        ignore_tables = dst_partner._context.get('ignore_tables_fk', False)
```

Comment:
- It's not clear we're into a partner merging context by reading context key
- Superfluous default value for `get()`

Reply:
Done